core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -2023,6 +2023,22 @@ package object config {
       .stringConf
       .createWithDefaultString("PWR")
 
+  private[spark] val EXECUTOR_DECOMMISSION_BATCH_INTERVAL =
+    ConfigBuilder("spark.executor.decommission.batchInterval")
+      .doc("Executors are decommissioned in batches to avoid overloading network bandwidth in" +
+        " migrating rdd and shuffle data. This config sets the interval between batches.")
+      .version("3.2.0")
+      .timeConf(TimeUnit.MILLISECONDS)
+      .createWithDefault(3000)
+
+  private[spark] val EXECUTOR_DECOMMISSION_BATCH_SIZE =
+    ConfigBuilder("spark.executor.decommission.batchSize")
+      .doc("Executors are decommissioned in batches to avoid overloading network bandwidth in" +
+        " migrating rdd and shuffle data. This config sets the size of a batch.")
+      .version("3.2.0")
```

Comment:
Let's change this to 3.3.0 since this is now too late for 3.2 (sorry for not reviewing this sooner).